### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/cookbook/tutorial/command.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#587] [Announcements]
# https://theme-hope.vuejs.press/zh/cookbook/tutorial/command.html

本教程介绍 VuePress 项目命令。

https://theme-hope.vuejs.press/zh/cookbook/tutorial/command.html

**Comment by XiaoYang943** [comment]
赞👍
